src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,142 @@
+package seedu.addressbook.common;
+
+import static org.junit.Assert.*;
+
+import java.util.ArrayList;
+import java.util.Collection;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class UtilsTest {
+	
+	@Before
+	public void setUp() {
+		;
+	}
+	
+	/* 
+	 * Tests for Utils.isAnyNull ====================
+	*/
+	
+	@Test
+	public void isAnyNull_passInEmptyInput() {
```

Comment:
Note the naming convention for test methods (the first part is feature/method under test), e.g. `isAnyNull_emptyList_falseReturned`